src/community/netcdf-ghrsst/src/test/java/org/geoserver/wcs/responses/GHRSSTWCSTest.java

```diff
@@ -0,0 +1,218 @@
+
+        // hope the file name structure is correct, not 100% sure 
+        String contentDispostion = response.getHeader("Content-disposition");
+        assertEquals("inline; filename=19121213214553-EUR-L3U_GHRSST-SSTint-AVHRR_METOP_A-v02.0-fv01.0.nc", 
+                contentDispostion);
+        byte[] responseBytes = getBinary(response);
+        File file = File.createTempFile("ghrsst", ".nc", new File("./target"));
+        FileUtils.writeByteArrayToFile(file, responseBytes);
+        try (NetcdfDataset dataset = NetcdfDataset.openDataset(file.getAbsolutePath(), true, null)) {
+            assertNotNull(dataset);
+
+            // check global attributes
+            Map<String, Attribute> globalAttributes = getGlobalAttributeMap(dataset);
+            assertNotNull(globalAttributes.get("uuid"));
+            UUID.fromString(globalAttributes.get("uuid").getStringValue());
+            assertNotNull(globalAttributes.get("date_created"));
+            assertNotNull(globalAttributes.get("spatial_resolution"));
+            // input test file is a freak that really has this resolution, as verified with gdalinfo
+            // e.g. gdalinfo NETCDF:"sst-orbit053.nc":"sea_surface_temperature"
+            assertEquals("179.9499969482422 degrees", globalAttributes.get("spatial_resolution").getStringValue());
+            // likewise, it really has time time
+            assertNotNull(globalAttributes.get("start_time"));
+            assertNotNull(globalAttributes.get("time_coverage_start"));
+            assertEquals("19121213T204553Z", globalAttributes.get("start_time").getStringValue());
+            assertEquals("19121213T204553Z", globalAttributes.get("time_coverage_start").getStringValue());
+            assertNotNull(globalAttributes.get("stop_time"));
+            assertNotNull(globalAttributes.get("time_coverage_end"));
+            assertEquals("19121213T204553Z", globalAttributes.get("stop_time").getStringValue());
+            assertEquals("19121213T204553Z", globalAttributes.get("time_coverage_end").getStringValue());
+            // and these bounds
+            double EPS = 1e-3;
+            assertNotNull(globalAttributes.get("northernmost_latitude"));
+            assertNotNull(globalAttributes.get("southernmost_latitude"));
+            assertNotNull(globalAttributes.get("westernmost_longitude"));
+            assertNotNull(globalAttributes.get("easternmost_longitude"));
+            assertEquals(119.925, globalAttributes.get("northernmost_latitude").getNumericValue().doubleValue(), EPS);
+            assertEquals(-119.925, globalAttributes.get("southernmost_latitude").getNumericValue().doubleValue(), EPS);
+            assertEquals(-269.925, globalAttributes.get("westernmost_longitude").getNumericValue().doubleValue(), EPS);
+            assertEquals(269.925, globalAttributes.get("easternmost_longitude").getNumericValue().doubleValue(), EPS);
+            // resolution, take 2
```

Comment:
Are these lat and lon correct? lat +/-119 ?

Reply:
As the comment says, it really has those bounds... it's a reduced version of a larger netcdf, compression the resolution so much I guess affected the bounds. But it's a match for gdalinfo output